.github/workflows/test-unit.yml.example

```diff
@@ -0,0 +1,34 @@
+# Github Action work-flow definition, copy to .github/workflows/
+#
+#   `mkdir -p .github/workflows && cp ci/.github/*.yml .github/workflows/`
+#
+name: Test Unit
+
+on: [push, pull_request]
+
+jobs:
+
+  test:
+    name: Elastic Stack ${{ matrix.es-version }} on ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        es-version: [6.x, 7.x]
+        os: [ubuntu-latest]
+
+    runs-on: ${{ matrix.os }}
+
+    env:
+      ELASTIC_STACK_VERSION: ${{ matrix.es-version }}
+
+    steps:
+      - uses: actions/checkout@v1
+
```

Comment:
Could we actually make the submodule setup part of the workflow too?
1. git submodule add
2. symlink .ci to ci

Reply:
that is something you do when you setup the .ci plugin as a sub-module on the plugin
here we're already assuming .ci sub-module exists and later `git submodule update --init`